### PR TITLE
Add support pagination to getRelations

### PR DESCRIPTION
**Pull request by Sergey-weber**

### CODE REVIEW

File: src/services/entry/actions/get-related-entries.ts [hunk 1]
```diff
@@ -82,14 +89,28 @@ export async function getRelatedEntries(
...
-        .orderBy('depth')
-        .timeout(
-            extendedTimeout ? EXTENDED_QUERY_TIMEOUT : DEFAULT_QUERY_TIMEOUT,
-        )) as unknown[] as GetRelatedEntriesResult[];
+        .where((builder) => {
+            if (scope) {
+                builder.where({[EntryColumn.Scope]: scope});
+            }
+        })
+        .orderBy('depth');
```
Comment (jhoncool) [inline src/services/entry/actions/get-related-entries.ts:97]: Let's change 'depth' to 'createdAt'

**Review by jhoncool — DISMISSED** [review]

File: src/services/entry/actions/get-entry-relations.ts [hunk 2]
```diff
@@ -138,5 +180,13 @@ export async function getEntryRelations(
...
 
+    // TODO: leave a response with pagination only, when there will be pagination support everywhere in the frontend
+    if (isPagination) {
+        return {
+            relations,
+            nextPageToken,
+        };
+    }
+
     return relations;
```
Comment (jhoncool) [inline src/services/entry/actions/get-entry-relations.ts:191]: Let's return here always object
```
        return {
            relations,
            nextPageToken,
        };
```

if `isPagination` is false then `nextPageToken` is always undefined

Comment (Sergey-weber) [reply src/services/entry/actions/get-entry-relations.ts:191]: we need to support the contract that was before, the frontend is not expecting the object now, there will be an error, there are many places on the client where this handle is used

Comment (jhoncool) [reply src/services/entry/actions/get-entry-relations.ts:191]: Yes but this function doesn't use direct in api

**Review by jhoncool — APPROVED** [review]

**Pull request merged**